Assets/Mapzen/Unity/Editor/MapStyleEditor.cs

```diff
@@ -41,6 +44,16 @@ public override void OnInspectorGUI()
                 return;
             }
 
+            GUILayout.Label("Style hot reloading", labelBoldStyle);
+
+            previewMode = GUILayout.Toggle(previewMode, "Preview mode");
```

Comment:
Let's just describe this option right on the button: "Update RegionMap while editing", then remove the space and label below this.